cmake/modules/FindWaylandpp.cmake

```diff
@@ -37,6 +37,34 @@ find_library(WAYLANDPP_CURSOR_LIBRARY NAMES wayland-cursor++
 find_library(WAYLANDPP_EGL NAMES wayland-egl++
                            PATHS ${PC_WAYLANDPP_LIBRARY_DIRS})
 
+if(KODI_DEPENDSBUILD)
+  pkg_check_modules(PC_WAYLANDC wayland-client wayland-egl wayland-cursor QUIET)
+
+  if("webos" IN_LIST CORE_PLATFORM_NAME_LC)
+    # prefer wayland libs from toolchain
+    set(WAYLAND_SEARCH_PATH ${TOOLCHAIN}/${HOST}/sysroot/usr/lib)
```

Comment:
It would be nice to factor this out to not need platform specifics in the modules.